packages/gatsby-theme-docs/src/layouts/internals/sidebar.js

```diff
@@ -354,9 +355,12 @@ const Sidebar = (props) => {
         <WebsiteTitle theme={theme}>
           <SpacingsStack scale="xs">
             <div>{props.isGlobalBeta && <BetaFlag />}</div>
-            <WebsiteTitleLink as={Link} id="site-title" to="/" theme={theme}>
-              {props.siteTitle}
-            </WebsiteTitleLink>
+            <SpacingsInline scale="s">
+              <PlaceholderSiteIcon />
```

Comment:
you could try if it works too when being part of the clickable link (should ideally behave like the top site level icon+text).  But don't invest a lot.